.changeset/generateLineageChangeSet.md

```diff
@@ -0,0 +1,7 @@
+---
+"@finos/legend-application-studio": patch
+"@finos/legend-application": patch
+"@finos/legend-graph": patch
+---
+
+Adding Optional Service Lineage Feature to Registration
```

Comment:
You did not make any change to `@finos/legend-application` therefore it should not be in your changeset